crates/subspace-farmer/src/plot.rs

```diff
@@ -322,21 +245,26 @@ impl Plot {
         }
     }
 
-    pub fn read_piece(&self, piece_index: u64) -> io::Result<Vec<u8>> {
-        self.read_pieces(piece_index, 1)
+    pub fn read_piece(&self, index: PieceIndex) -> io::Result<Vec<u8>> {
+        self.read(index)
+            .map(|piece| <[u8; PIECE_SIZE]>::from(piece).to_vec())
     }
 
     // TODO: Replace index with offset
     /// Returns pieces packed one after another in contiguous `Vec<u8>`
-    pub(crate) fn read_pieces(&self, first_index: u64, count: u64) -> io::Result<Vec<u8>> {
+    pub(crate) fn read_pieces(
+        &self,
+        cursor_offset: u64,
```

Comment:
`cursor_offset` is not super clear compared to `first_index` especially when you put `read_piece` and `read_pieces` altogether.

- Before: `read_piece(index)`, `read_pieces(first_index, count)`.
- Now: `read_piece(index)`, `read_pieces(cursor_offset, count)`.

Reply:
Should I make type alias here? Like:
```rust
/// Index of piece on disk
type PieceCursorOffset = u64;
```

Reply:
`cursor_offset` exposes internal implementation (even though there is no external cursor concept in the protocol), `piece_offset` variable that you changed it to is much better since that is the terminology we use for plot organization (it is also implementation detail, but consistent with the rest of the codebase).